### PR TITLE
Kernels 2021-06-03

### PR DESCRIPTION
**Pull request by ymatsiuk**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ymatsiuk** [comment]
@GrahamcOfBorg test kernel-latest kernel-lts latestKernel.login latestKernel.hardened

**Comment by mweinelt** [comment]
Waiting for hardened patch updates.

**Comment by Ma27** [comment]
Also, I'm wondering if we should remove 5.11 now that it's EOL.

**Comment by mweinelt** [comment]
> Also, I'm wondering if we should remove 5.11 now that it's EOL.

Yes, please.

**Comment by ymatsiuk** [comment]
@Ma27 @mweinelt https://github.com/NixOS/nixpkgs/pull/125469/commits/95ae54fed869a2d4c2ba4e4412c15ff43febb328 removes `5.11` (not sure if I did it right though, so please check this out)

**Comment by ymatsiuk** [comment]
@GrahamcOfBorg test kernel-latest kernel-lts latestKernel.login latestKernel.hardened

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -20801,9 +20786,7 @@ in
   # Realtime kernel packages.
   linuxPackages-rt_5_4 = linuxPackagesFor pkgs.linux-rt_5_4;
   linuxPackages-rt_5_10 = linuxPackagesFor pkgs.linux-rt_5_10;
-  linuxPackages-rt_5_11 = linuxPackagesFor pkgs.linux-rt_5_11;
   linuxPackages-rt = linuxPackages-rt_5_4;
-  linuxPackages-rt_latest = linuxPackages-rt_5_11;
   linux-rt = linuxPackages-rt.kernel;
   linux-rt_latest = linuxPackages-rt_latest.kernel;
```
Comment (fabianhjr) [inline pkgs/top-level/all-packages.nix:20791]: While `linuxPackages-rt_5_11` is going to be removed alongside 5.11 I don't think `linuxPackages-rt_latest` should be removed, it should be repointed to the latest version either 5.10 or 5.12?

Comment (fabianhjr) [reply pkgs/top-level/all-packages.nix:20791]: Another possibility could be to delay 5.11 removal until linux-rt-5.12 gets released

Comment (ymatsiuk) [reply pkgs/top-level/all-packages.nix]: OMG! Did I do this? 😊 Let me fix this quickly before anyone noticed 🙂  Thanks!

Comment (ymatsiuk) [reply pkgs/top-level/all-packages.nix]: Should be fixed now. Was in a hurry yesterday. Thanks for the review @fabianhjr!

**Comment by mweinelt** [comment]
@GrahamcOfBorg eval

**Comment by mweinelt** [comment]
```
while evaluating the attribute 'src' of the derivation 'linux-config-5.12.9' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-1/pkgs/stdenv/generic/make-derivation.nix:201:11:
hash '' has wrong length for hash type 'sha256'
```

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (fabianhjr) [reply pkgs/top-level/all-packages.nix]: You are welcome, thanks for pushing the Kernel Updates along :3

**Comment by ymatsiuk** [comment]
@GrahamcOfBorg test kernel-latest kernel-lts latestKernel.login latestKernel.hardened

**Comment by Ma27** [comment]
FYI I fixed the evaluation of the `kernel-generic` test in 10eab5b6b3d1d38ffd3594fa6e4be13924dafd15.

**Pull request merged**